auth-manager/src/main/java/com/qpay/authmanager/controller/AuthController.java

```diff
@@ -0,0 +1,29 @@
+package com.qpay.authmanager.controller;
+
+import com.qpay.authmanager.model.dto.UserModification;
+import com.qpay.authmanager.service.AuthenticationService;
+import com.qpay.authmanager.utility.PathUtils;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(path = PathUtils.AUTH_PATH)
+public class AuthController {
+
+    private final AuthenticationService authenticationService;
+
+    @PostMapping("/signIn")
+    public String singIn(@RequestBody final UserModification userModification) {
```

Comment:
```suggestion
    public JwtAuthenticationResponse signIn(@RequestBody final AuthCredentials authCredentials) {
```

public record UserCredentials(String email, String password) { }

public record JwtAuthenticationResponse(String token) { }